containers/Balancer.ts

```diff
@@ -104,14 +71,12 @@ const useBalancer = () => {
     skip: !selectedTokenAddress,
     context: { clientName: "BALANCER" },
     variables: { tokenId: selectedTokenAddress },
-    pollInterval: 5000,
```

Comment:
This removes the polling, this might be nice in the future but I don't see many user complaints about non-refreshing data.

Reply:
does this mean the only way to update the table is to hard reload it?

Reply:
I think so, I'm OK with this though as its a UI tradeoff for performance